File: Top 50 Coding Interview Questions/28. Write a code to find Find the ASCII value of a character/ASCIIvalueofacharacter.cpp
#include<iostream> // Include the iostream library for input and output operations
using namespace std; // Use the standard namespace to avoid prefixing std::

int main() {
    char val; // Declare a variable to store the character input by the user

    // Prompt the user to enter a character
    cout << "Enter a character: ";
    cin >> val; // Read the character input from the user

    // Printing the ASCII value of the input character
    // Typecasting the character 'val' to an integer to get its ASCII value
    cout << "The ASCII value of '" << val << "' is " << (int)val;

    return 0; // Return 0 to indicate successful execution
}

File: Top 50 Coding Interview Questions/28. Write a code to find Find the ASCII value of a character/ASCIIvalueofacharacter.py
# Python program to calculate the ASCII value of a character

# Prompt the user to enter a character
val = input("Enter a character: ")

# Check if the input is a single character
if len(val) == 1:
    # Use the ord() function to get the ASCII value of the character
    print(f"The ASCII value of '{val}' is {ord(val)}")
else:
    print("Please enter a single character.")
